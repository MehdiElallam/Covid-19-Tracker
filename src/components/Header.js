import React from 'react'
import styled from 'styled-components';


export default function Header({title}) {
    return (
        <Wrapper>
            <h1>{title}</h1>
            <p>The official UK government website for data and insights on coronavirus (COVID-19).</p>
        </Wrapper>
    )
}

const Wrapper = styled.div`
    background: #000;
    width: 100%;
    margin-bottom : 20px;
    padding: 15px 25px;
    color: #FFF;
`;