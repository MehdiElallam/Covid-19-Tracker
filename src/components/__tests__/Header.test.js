import React from 'react';
import { configure, shallow } from 'enzyme';
import Header from '../Header';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('Header', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Header debug />);
  
    expect(component).toMatchSnapshot();
  });
});