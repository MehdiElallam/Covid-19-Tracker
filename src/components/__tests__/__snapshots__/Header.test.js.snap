// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
